CleanSwift/CleanSwift/Scene/DetailNewsScene/DetailNewsInteractor.swift

```diff
@@ -0,0 +1,35 @@
+// DetailNewsInteractor.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import UIKit
+
+/// Интерактор экрана с новостью
+final class DetailNewsInteractor {
+    private var presenter: DetailNewsPresentationLogic?
+    private var worker: DetailNewsWorkerBusinessLogic?
+
+    var data: Data?
+
+    init(presenter: DetailNewsPresentationLogic?, worker: DetailNewsWorkerBusinessLogic?) {
```

Comment:
```suggestion
    private var worker: DetailNewsWorkerBusinessLogic?

    var data: Data?

    init(presenter: DetailNewsPresentationLogic?, worker: DetailNewsWorkerBusinessLogic?) {
```
очередность (private, public) и марки